packages/react-scripts/config/webpack.config.js

```diff
@@ -33,13 +33,23 @@ const ModuleNotFoundPlugin = require('react-dev-utils/ModuleNotFoundPlugin');
 const ForkTsCheckerWebpackPlugin = require('react-dev-utils/ForkTsCheckerWebpackPlugin');
 const typescriptFormatter = require('react-dev-utils/typescriptFormatter');
 // @remove-on-eject-begin
-const eslint = require('eslint');
+// const eslint = require('eslint');
 const getCacheIdentifier = require('react-dev-utils/getCacheIdentifier');
 // @remove-on-eject-end
 const postcssNormalize = require('postcss-normalize');
-
 const appPackageJson = require(paths.appPackageJson);
 
+const sassFunctions = require('bpk-mixins/sass-functions');
+const camelCase = require('lodash/camelCase');
+const bpkReactScriptsConfig = appPackageJson['backpack-react-scripts'] || {};
+const customModuleRegexes = bpkReactScriptsConfig.babelIncludePrefixes
+  ? bpkReactScriptsConfig.babelIncludePrefixes.map(
+      prefix => new RegExp(`node_modules[\\/]${prefix}`)
+    )
+  : [];
+const cssModulesEnabled = bpkReactScriptsConfig.cssModules !== false;
```

Comment:
Check if consumers are setting to false to see if this can be deprecated

Reply:
Did a check and there is use of this config variable being set to false so will need to keep this

Reply:
Ok